saas/{{cookiecutter.project_name}}/webapp/src/lib/components/forms/form.svelte

```diff
@@ -110,7 +110,7 @@
 
 	{#if useDefaultSubmitButton}
 		<div class="flex justify-end">
-			<Button type="submit" disabled={hasErrors}>{defaultSubmitButtonText}</Button>
+			<Button id="submit" type="submit" disabled={hasErrors}>{defaultSubmitButtonText}</Button>
```

Comment:
does this prevent more that one form in a page?

Reply:
I'm not sure, but I guess it can.
If we had two forms on the same page, there would be two elements with the same id, which is not allowed by html.

To solve this properly, I think we could always pass the submit button explicitly to the form, giving it its own id.
We could make a `<SubmitButton/>` component.

This also reduces the props needed for the `<Form />` component and allows for more control by the user.